2_adventures/001/blndcat/06_day6/ruby-app/spec/app_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'app'
+
+describe App do
+	it "should launch" do
+		expect(App.launch).to be true
+	end
+end
```

Comment:
Thanks Victor, I never realised that I was missing the expandtab option in my vimrc file. One time when even the almightly StackOverflow was wrong, or at least the highest rated marked, as accepted was.  

http://stackoverflow.com/questions/1562633/setting-vim-whitespace-preferences-by-filetype

The funny thing is that it was asked by Drew Neil of Vimcasts fame, albeit back in 2009.